deploy-commerce-engine.ps1

```diff
@@ -85,8 +85,8 @@ Function  Start-CommerceEnginePepare ( [string] $basePublishPath = $(Join-Path $
     $idServerSettings | ConvertTo-Json -Depth 10 -Compress | set-content $idServerJson
 
     #Modify BizFx to match new hostname
-    $bizFxJson = $([System.IO.Path]::Combine($webRoot, $BizFxPathName, "assets\config.json")
-    $bizFxSettings = $bizFxJson | Get-Content $pathToJson -Raw | ConvertFrom-Json)
+    $bizFxJson = $([System.IO.Path]::Combine($webRoot, $BizFxPathName, "assets\config.json"))
+    $bizFxSettings = Get-Content $bizFxJson -Raw | ConvertFrom-Json
```

Comment:
@jeanfrancoislarente all updates in this file are for solving the following issues
![hh902_5](https://user-images.githubusercontent.com/6151021/43677044-edefb844-9825-11e8-89c5-8d5cffd95ff4.png)